templates/packer/ansible/roles/common/tasks/packages.yml

```diff
@@ -106,16 +120,16 @@
 
 - name: Install worker rpm
   tags:
-    - rpmcopy
-    - rpmrepo_osbuild
+    - ci
```

Comment:
I'm just syntactically checking...
if this was skipped for `rpmcopy` or `rpmrepo_osbuild` before - it would have been never run?

Reply:
indeed